sys/arm64/arm64/db_trace.c

```diff
@@ -94,6 +94,14 @@ db_stack_trace_cmd(struct thread *td, struct unwind_state *frame)
 			struct trapframe *tf;
 
 			tf = (struct trapframe *)(uintptr_t)frame->fp - 1;
+#ifdef __CHERI_PURE_CAPABILITY__
+			if (!cheri_can_access(tf,
+			    CHERI_PERM_LOAD | CHERI_PERM_LOAD_CAP,
+			    (vm_offset_t)tf, sizeof(*tf))) {
```

Comment:
s/vm_offset_t/ptraddr_t/

Same underlying type and meaning but the prototype does say ptraddr_t for cheri_can_access